benchmark/BDN.benchmark/Program.cs

```diff
@@ -32,14 +33,21 @@ public BaseConfig()
 
         var baseJob = Job.Default.WithGcServer(true);
 
-        Net8BaseJob = baseJob.WithRuntime(CoreRuntime.Core80)
-            .WithEnvironmentVariables(new EnvironmentVariable("DOTNET_TieredPGO", "0"));
-        Net9BaseJob = baseJob.WithRuntime(CoreRuntime.Core90)
-            .WithEnvironmentVariables(new EnvironmentVariable("DOTNET_TieredPGO", "0"));
-
-        AddJob(
-            Net8BaseJob.WithId(".NET 8"),
-            Net9BaseJob.WithId(".NET 9")
-            );
+        var runtime = System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription;
+        switch (runtime)
```

Comment:
Let the user specify this as command line parameter. If they didn't specify anything then run both frameworks so we are backward compatible with main.